SU-SWS/stanford_profile_helper
# D8core-2765 Styles and icons for mailto buttons

# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- fixes mailto links on buttons
https://stanfordits.atlassian.net/browse/D8CORE-2765

# Needed By (Date)
soon

# Urgency
Not urgent

# Steps to Test

1. Check out this branch and clear cache
2. create buttons with with mailto links. There are 3 types of buttons.
3. Verify appropriate color of mailto link icon


# Affected Projects or Products
-  D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-2765
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@ishahism 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
